src/core/Directus/Database/Schema/Object/Field.php

```diff
@@ -491,6 +491,16 @@ public function isOneToMany()
     {
         return $this->hasRelationship() ? $this->getRelationship()->isOneToMany() : false;
     }
+    
+    /**
+     * Checks whether the relationship is MANY TO MANY
+     *
+     * @return bool
+     */
+    public function isManyToMany()
+    {
+        return $this->hasRelationship() && $this->getRelationship()->isOneToMany() && $this->getInterface() == "many-to-many" ? true : false;        
```

Comment:
@benhaynes @rijkvanzanten  Is that okay if we implement the condition based on the interface? 

I think here we can implement the condition based on the `directus_relations` table instead of the interface. Cause for M2M; we have 2 entries of a junction table in `directus_relations` table and for the others, there is only single entry!

Reply:
I leave this decision to Rijk,  but that makes sense to me.

Reply:
We can not filter based on the name of the interface. For example, `files` is also a m2m style interface, and there will most likely be more in the future

Reply:
Okay. As all of us are agreed that;

> We can not filter based on the name of the interface. 

So @itsmerhp kindly replace the condition of interface with relations table entry! Let me know if you need any help for the logic!

Reply:
@benhaynes @rijkvanzanten @bjgajjar 
I have applied required changes in PR. Please check it.